fbchat/models.py

```diff
@@ -267,10 +267,20 @@ def __init__(self, url=None, size=None, name=None, is_malicious=None, **kwargs):
         self.name = name
         self.is_malicious = is_malicious
 
-class AudioAttachment(FileAttachment):
-    def __init__(self, **kwargs):
-        """Represents an audio file that has been sent as a Facebook attachment - *Currently Incomplete!*"""
-        super(StickerAttachment, self).__init__(**kwargs)
+class AudioAttachment(Attachment):
+    # Name of the file
```

Comment:
These comments should start with a colon, like this: `#: comment`, so they'll show up in the sphinx autogenerated docs on [ReadTheDocs](http://fbchat.readthedocs.io/)